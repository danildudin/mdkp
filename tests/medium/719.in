29 6 0
420 814 860 977 492 91 718 814 147 7 262 213 621 325 864 681 847 776 359 297 994 651 73 792 505 432 140 776 154 
162 725 98 111 47 401 948 817 21 66 125 811 4 842 806 683 712 18 593 642 542 731 521 327 612 678 89 710 380 
948 824 508 904 897 548 501 126 915 599 441 290 480 514 93 522 487 35 741 964 446 788 670 529 725 359 845 579 694 
447 110 643 504 592 142 78 315 877 160 285 223 630 784 778 367 973 587 762 23 651 66 148 164 992 257 837 99 795 
968 869 659 143 289 680 19 645 988 260 861 805 57 986 123 970 941 569 675 439 329 321 305 861 604 402 445 810 949 
962 996 780 235 351 99 321 467 768 283 147 5 507 825 337 759 296 958 723 935 951 174 547 941 891 25 42 132 107 
629 24 237 802 665 503 654 697 778 449 81 142 988 687 556 270 587 735 357 687 331 478 389 198 367 759 546 125 240 
7356 8346 8614 6883 5437 5681 
