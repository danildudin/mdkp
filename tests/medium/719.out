9282
13
1 2 3 4 6 7 10 11 14 16 20 21 27 
